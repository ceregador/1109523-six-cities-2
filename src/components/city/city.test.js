import React from 'react';
import Renderer from 'react-test-renderer';
import City from '../city/city.jsx';

it(`Inactive city renders correctly`, () => {
  const tree = Renderer
    .create(
        <City
          city={{name: ``}}
          isActive={false}
          onClick={() => null}
        />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it(`Active city renders correctly`, () => {
  const tree = Renderer
    .create(
        <City
          city={{name: ``}}
          isActive={true}
          onClick={() => null}
        />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
