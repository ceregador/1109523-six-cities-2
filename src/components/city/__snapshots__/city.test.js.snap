// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Active city renders correctly 1`] = `
<li
  className="locations__item"
  onClick={[Function]}
>
  <a
    className="locations__item-link tabs__item tabs__item--active"
    href="#"
  >
    <span>
      
    </span>
  </a>
</li>
`;

exports[`Inactive city renders correctly 1`] = `
<li
  className="locations__item"
  onClick={[Function]}
>
  <a
    className="locations__item-link tabs__item"
    href="#"
  >
    <span>
      
    </span>
  </a>
</li>
`;
